setup.py

```diff
@@ -17,7 +17,7 @@
 sqlite_requires = ['sqlalchemy']
 postgres_requires = sqlite_requires + ['psycopg2']
 mysql_requires = sqlite_requires + ['pymysql']
-mapd_requires = ['pymapd>=0.8.2']
+mapd_requires = ['pymapd>=0.12.0']
```

Comment:
This may require some fiddling with python version specific requirements in the call to `setup` below. See https://setuptools.readthedocs.io/en/latest/setuptools.html?highlight=environment%20markers#declaring-platform-specific-dependencies for details on how to do that.

Alternatively, drop support for Python 3.5 for the omnisci backend.

Reply:
Yeah, this will take a bit of work to nail down, I'll think on it a bit.

Reply:
A slightly simpler way might be to use `sys.version_info` and set the value of `mapd_requires` based on the running interpreter version.